tests/foreman/installer/test_installer.py

```diff
@@ -1343,19 +1343,29 @@ def common_sat_install_assertions(satellite):
     result = satellite.execute(r'grep -iR "error" /var/log/candlepin/*')
     assert len(result.stdout) == 0
 
+    httpd_log = satellite.execute('journalctl --unit=httpd')
+    assert "WARNING" not in httpd_log.stdout
+
     result = satellite.cli.Health.check()
     assert 'FAIL' not in result.stdout
 
 
-def install_satellite(satellite, installer_args):
+def install_satellite(satellite, installer_args, enable_fapolicyd=False):
     # Register for RHEL8 repos, get Ohsnap repofile, and enable and download satellite
     satellite.register_to_cdn()
     satellite.download_repofile(
         product='satellite',
         release=settings.server.version.release,
         snap=settings.server.version.snap,
     )
+    if enable_fapolicyd:
+        satellite.execute(
+            'dnf -y install fapolicyd && systemctl enable --now fapolicyd'
+        ).status == 0
     satellite.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
+    if enable_fapolicyd:
+        assert satellite.execute('rpm -q foreman-fapolicyd').status == 0
+        assert satellite.execute('rpm -q foreman-proxy-fapolicyd').status == 0
```

Comment:
Why was this block not added in the previous block where you installed the packages ?

Reply:
Because `foreman-fapolicyd` and `foreman-proxy-fapolicyd` would be installed if the `fapolicyd` is installed before `satellite` package is installed that's why the assertion is here.